_maps/configs/engineeringmess.json

```diff
@@ -0,0 +1,37 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"prefix": "ISV",
+	"namelists": ["GENERAL", "NATURAL", "ENGINEERING"],
+	"map_name": "Rube Goldberg-class Engineering Project",
+	"map_short_name": "Rube Goldberg-class",
+	"map_path": "_maps/shuttles/shiptest/engineeringmess.dmm",
+	"map_id": "engineeringmess",
+	"limit": 1,
+	"job_slots": {
+		"Chief at Engineering": {
+			"outfit": "/datum/outfit/job/ce",
+			"slots": 1
```

Comment:
"outfit": "/datum/outfit/job/ce",
"officer": true,
"slots": 1

there's a flag for officer positions. the first position serves as captain but adding an officer flag gives him the officer symbol on the manifest.

Reply:
Good catch.